pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -448,8 +452,8 @@ def get_sync_details(remote, url, sync_policy, repository):
             repomd_path = result.path
             repomd = cr.Repomd(repomd_path)
             repomd_checksum = get_sha256(repomd_path)
-            treinfo_file_data = get_treeinfo_data(remote, url)
-            treeinfo_checksum = treinfo_file_data.get("hash", "")
+            treeinfo_file_data = get_treeinfo_data(remote, url)
```

Comment:
Are we missing skip types as a trigger for full resync here?  I think we are.  It wouldn't impact the treeinfo because we have a separate field for that but I think it would affect SRPMs

Reply:
I'm not sure I understand what you're looking for here - can you expand a bit?

(Note that the only thing this PR changes in this area is the spelling of "treinfo")

Reply:
Right, this isn't a complaint about the PR as such, I just mean that our existing code might have a bug around optimizing sync when `skip_types` changes.